src/openms/include/OpenMS/FILTERING/TRANSFORMERS/SpectraMerger.h

```diff
@@ -436,6 +676,236 @@ namespace OpenMS
 
     }
 
+    /**
+     * @brief average spectra (profile mode)
```

Comment:
please explain how the profile mode function differes from the centroid mode function